### DISCUSSION TITLE
Throwing the cat among the pigeons: Craft CMS in Laravel

### DISCUSSION THREAD

**Discussion by dolphiq-allard** [craftcms/cms#15421] [General]
Hey guys,

Just going to be bold here and throw the cat among the pigeons. Since we are a Laravel orientated digital agency, I am having a hard time getting the team to love Craft CMS. Me, I am a Craft CMS fanboy; the way the CMS works is absolutely stunning. The way Brandon and his team follow up on suggestions and tips is unheard of.

If only, if only it would be in Laravel, we, and probably many developers along with us, would be able to incorporate it into to our digital solutions so much easier. And my beloved team members would also be able to feel the power of this CMS.

How realistic would it be to see Craft CMS powered by the Laravel framework one day? The impact on both the Craft team and all plugin developers is massive, so this is a real challenging issue.

I know porting software from one framework to another is a heck of a job, but maybe with the power of AI and the dedicated community, this slowly becomes in reach. My team and I, for one would be more than willing to dedicate some knowledge and time to see this happening.

Really looking forward hearing from both the Craft team as well as fellow Craft CMS fans out there on how you guys look at this.

Just letting my thoughts flow here and hoping for an interesting discussion...

Cheers

**Comment by thupsi** [comment]
I have good news: P&T has already [announced the transition to Laravel](https://craftcms.com/blog/change-of-plans#:~:text=Dot%20All%20Lisbon%20will%20now%20take%20place%20September%2023%E2%80%9325%2C%202025%2C%20where%20we%E2%80%99ll%20announce%20Craft%206%20and%20kick%20off%20our%20transition%20to%20Laravel.). No details and exact timeframe have been shared though.

**Reply by dolphiq-allard** [reply]
Must have missed that one, but great news! Looking forward to this transition.

**Reply by vitalijalbu** [reply]
But from which version?😁
I remember they said that v5 will be based on laravel, but nothing…

**Reply by thupsi** [reply]
@vitalijalbu See the link above. The transition is planned to start with v6 but further details are scarce at this point, so take this with a grain of salt.
